test/metric.jl

```diff
@@ -43,7 +43,7 @@ Manifolds.injectivity_radius(::BaseManifold) = Inf
 Manifolds.local_metric(::MetricManifold{BaseManifold{N},BaseManifoldMetric{N}},x) where N = 2*one(x*x')
 Manifolds.exp!(M::MetricManifold{BaseManifold{N},BaseManifoldMetric{N}}, y, x, v) where N = exp!(base_manifold(M), y, x, v)
 Manifolds.vector_transport_to!(::BaseManifold, vto, x, v, y, ::ParallelTransport) = (vto .= v)
-Manifolds.is_default_metric(::BaseManifold,::DefaultBaseManifoldMetric) = Val(true)
+Manifolds.is_default_metric(::BaseManifold,::DefaultBaseManifoldMetric)
```

Comment:
This line does nothing now? Or am i missing something?

Reply:
ups, see last comment. I might have removed that because I thought it was a test...to answer your question: Yes, this line does nothing now. Your not missing something – I did. ;)